### PR TITLE
Improve project overview description & general site

### PR DESCRIPTION
**Pull request by sarahmonster**
This got a bit away from me a wee bit, but it's basically a bunch of meta improvements to the README as well as to the actual Nautilus site itself.

Screenshot 2019-06-21 at 14 51 31

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ module.exports = {
       ],
     },
     {
-      name: '🎨 Form',
+      name: 'Form',
```
Comment (tofumatt) [inline styleguide.config.js]: I relunctantly admit this is a good call. It's cute but there are even accessibility issues using emojis in important text so free-form (without the right markup). Plus it makes linking weird.

Semi-related: we'll need an `<Emoji>` component! 😆

File: styleguide/components/ComponentsList/index.js [hunk 2]
```diff
@@ -43,16 +43,19 @@ export function ComponentsList({ classes, items }) {
                 css={css`
                   color: ${theme.colors.neutral.grey800} !important;
                   width: 100% !important;
-                  padding: 16px 0 !important;
+                  padding: ${theme.spacing.padding.m} 0 !important;
                   border-top: 2px solid !important;
                   display: block !important;
+                  text-transform: uppercase !important;
```
Comment (tofumatt) [inline styleguide/components/ComponentsList/index.js]: Brute-forced, eh?

File: README.md [hunk 3]
```diff
@@ -77,7 +100,11 @@ Please read (`CONTRIBUTING.md`) for details on our code of conduct and the proce
 
 ## Versioning
 
+<<<<<<< HEAD
 We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [releases page](https://github.com/octopusthink/nautilus/releases).
+=======
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/octopusthink/nautilus/tags).
```
[the remark this answers is not in the document]
Comment (sarahmonster) [reply README.md]: I rebased and this should be fixed, I think?

[the remark this answers is not in the document]
Comment (tofumatt) [reply README.md]: Hmm, nope; it's still in there.

File: styleguide.config.js (hunk 1 above)
Comment (sarahmonster) [reply styleguide.config.js]: Yeah, it looked super garbagey too. Once we have an icon component we can implement icons for these sections that work better stylistically, but I think the emojis, whilst cute, weren't doing us any favours in terms of either aesthetics or accessibility. (And you're right, a screen reader would read these out, which is totally unnecessary!)

File: styleguide/components/ComponentsList/index.js (hunk 2 above)
Comment (sarahmonster) [reply styleguide/components/ComponentsList/index.js]: Yeah, we might need to rebuild the ComponentList component so we don't need to do this nonsense, but for now I'm !important flagging like it's going out of style.

File: README.md [hunk 4]
```diff
@@ -77,7 +100,11 @@ Please read (`CONTRIBUTING.md`) for details on our code of conduct and the proce
 
 ## Versioning
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (sarahmonster) [reply README.md]: Fixed!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sarahmonster) [reply README.md]: I double-fixed it now and it's for sure fixed. Rawr.

**Pull request merged**